### DISCUSSION TITLE
Make Athens interoperable with other open-source knowledge based solutions such as Obsidian, Logseq, etc.

### DISCUSSION THREAD

**Discussion by jmartinezheras** [athensresearch/athens#761] [Feature Requests]
My guess is that many more people would use Athens Research as their primary knowledge based solution if they would know that they are not locked-in with the Athens data format.

I know it's for a good reason, that having a graph database really makes the difference in what things you can do compared to markdown files. However, having an option to export to Markdown format all the notes would be just amazing!

It would be great if the most popular knowledge based solution such as Athens, Obsidian, Logseq, Foam, etc. could get in touch with each other and find a way to make their user data interoperable. It could be as "simple" as agreeing in data exchange format.

Like this, we can rest assured that all the time we spend in Athens Research is well spent.

Note: I acknowledge that this might not be "simple" but it's worth the effort

**Comment by benjamindovinh** [comment]
Yes, exporting and importing data between other knowledge management projects would be awesome

**Reply by panomitrius** [reply]
I was about to add a request for markdown import/export - but this is a more comprehensive approach, which would be great. Perhaps a markdown import/export could be a start?

**Reply by VictorVow** [reply]
Being able to one-click import from WorkFlowy or Dynalist would be fantastic to have. They both use OPML as the data format.

**Comment by jsmorabito** [comment]
let's divide this into two discussions since i think theres a difference between interoperability between the software and then import/export markdown. @panomitrius can you make the markdown import/export discussion?

**Reply by panomitrius** [reply]
@jsmorabito Yes, sorry the delay, but now [done](https://github.com/athensresearch/athens/issues/920)!
